api/src/main/resources/messages.properties

```diff
@@ -3291,3 +3291,8 @@ OrderFrequency.concept.shouldBeClassFrequency=Concept should be of class Frequen
 OrderFrequency.concept.shouldNotBeShared=Only one OrderFrequency should be mapped to a particular concept
 
 Module.error.cyclicDependencies=Could not start some modules because they depend on others which are not started. Please first start them or use the "Start All" button on the "Manage Modules" page: {0}
+
+GivenName.startsOrEndsWithWhitespace=Invalid GivenName, Trailing or Leading Whitespace detected
```

Comment:
the error message should be something like "Given name may not start or end with blank spaces"